### DISCUSSION TITLE
2 ESP32s with BCR and LoRa to connect to SAP S/4HANA 😆

### DISCUSSION THREAD

**Discussion by skye0402** [toitlang/toit#235] [Show and tell]
Dear friends of Toit,

I completed my little setup and the blog yesterday:
![IMG_20211227_200829](https://user-images.githubusercontent.com/36907475/147747383-3ab893f8-fb00-4267-8af8-fdee31410660.jpg)
Basically I wanted to see how to integrate several components and do some web service calls. So here is the "remote barcode reader" 😄. Is it useful? Not sure, but it was fun to do. Video and Github is in the blog: https://medium.com/@techandfun/c61e52e67d86

Comments welcome!

**Comment by kasperl** [comment]
Thanks for writing this up! 👍
